core/components/atoms/page/Page.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { BaseProps, BaseHtmlProps } from '@/utils/types';
+import styles from '@css/components/page.module.css';
+
+export interface PageProps extends BaseProps, BaseHtmlProps<HTMLDivElement> {
+  /**
+   * Content to be rendered inside Page
+   */
+  children: React.ReactNode;
+  /**
+   * Sets custom padding for the Page
+   */
+  padding?: string;
```

Comment:
Please check instead of using `padding` as a prop we can use something like `size`, on the basis of which we apply the different css and not just padding in future